graphql-schemas/schema.yml

```diff
@@ -3217,6 +3218,30 @@ confs:
   - { name: origin, type: string, isRequired: true }
   - { name: records, type: DnsRecord_v1, isList: true }
 
+- name: CloudflareDnsRecord_v1
+  datafile: /cloudflare/dns-record-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: name, type: string, isRequired: true }
+  - { name: type, type: string, isRequired: true}
+  - { name: ttl, type: int , isRequired: true}
+  - { name: value, type: string , isRequired: true}
+  - { name: proxied, type: boolean }
+
+
+- name: CloudflareDnsZone_v1
+  datafile: /cloudflare/dns-zone-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: zone, type: string, isRequired: true }
+  - { name: partial, type: boolean }
```

Comment:
Is there a use case for `partial` when this configuration is intended to manage the DNS zone?

> A partial (CNAME) setup allows you to use Cloudflare’s reverse proxy while maintaining your primary and authoritative DNS provider.

Reply:
It would be best to match the field with what terraform expects for better clarify and less need to translate in code

Ex for the other Cloudflare work I had a `type` field enum with `partial` and `full` values. See https://github.com/app-sre/qontract-schemas/blob/400758fc32492fddc7770b1be7cbf40da8f8e0d7/schemas/cloudflare/terraform-resource-1.yml#L52-L55